### ISSUE TITLE
 what is the potential energy?

### ISSUE THREAD

**Issue by LinxuanHu** [ubsuny/23-Homework7G1#60]

**Comment by Pranjal-Srivastava-2023** [comment]
It should be around -28 units for all the configurations in figure 4.

**Comment by WildJimmy** [comment]
Yes, the professor said all should be around -28 except one is around -27

**Comment by yasmensarhan27** [comment]
> it that for all the 7 configurations?

yes

**Comment by pratibha77118** [comment]
Yes we I do get the potential energy value of  approx. -28.eV for both the configuration chosen.
